docs/collections.md

```diff
@@ -0,0 +1,138 @@
+# Collections
+Collections define the structure for the different content types on your static site. Since every site is different, the `collections` settings will be very different from one site to the next.
+
+Let's say your site has a blog, with the posts stored in `_posts/blog`, and files saved in a date-title format, like `1999-12-31-lets-party.md`. Each post begins with settings in yaml-formatted front matter, like so:
+
+``` yaml
+---
+layout: blog
+title: "Let's Party"
+date: 1999-12-31 11:59:59 -0800
+thumbnail: "/images/prince.jpg"
+rating: 5
+---
+
+This is the post body, where I write about our last chance to party before the Y2K bug destroys us all.
+```
+
+Given this example, our `collections` settings would look like this:
+
+``` yaml
+collections:
+  - name: "blog" # Used in routes, e.g., /admin/collections/blog
+    label: "Blog" # Used in the UI
+    folder: "_posts/blog" # The path to the folder where the documents are stored
+    create: true # Allow users to create new documents in this collection
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
+    fields: # The fields for each document, usually in front matter
+      - {label: "Layout", name: "layout", widget: "hidden", default: "blog"}
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Publish Date", name: "date", widget: "datetime"}
+      - {label: "Featured Image", name: "thumbnail", widget: "image"}
+      - {label: "Rating (scale of 1-5)", name: "rating", widget: "number"}
+      - {label: "Body", name: "body", widget: "markdown"}
+```
+
+Let's break that down:
+<table>
+  <tr>
+    <td><code>name</code></td>
+    <td>Post type identifier, used in routes. Must be unique.</td>
+  </tr>
+  <tr>
+    <td><code>label</code></td>
+    <td>What the post type will be called in the admin UI.</td>
+  </tr>
+  <tr>
+    <td><code>folder</code></td>
+    <td>Where files of this type are stored, relative to the repo root.</td>
+  </tr>
+  <tr>
+  <td><code><em>files</em></code></td>
+    <td>Where files of this type are stored, relative to the repo root. If you want to edit specific files, see the "Data Files" section below.</td>
+  </tr>
+  <tr>
+    <td><code>create</code></td>
+    <td>Set to <code>true</code> to allow users to create new files in this collection.
+</td>
+  </tr>
+  <tr>
+    <td><code>slug</code></td>
+    <td>Template for filenames. <code>{{year}}</code>, <code>{{month}}</code>, and <code>{{day}}</code> will pull from the post's <code>date</code> field or save date. <code>{{slug}}</code> is a url-safe version of the post's <code>title</code>. Default is simply <code>{{slug}}</code>.
+</td>
+  </tr>
+  <tr>
+    <td><code>fields</code></td>
+    <td>Fields listed here are shown as fields in the content editor, then saved as front matter at the beginning of the document (except for <code>body</code>, which follows the front matter). Each field contains the following properties:
```

Comment:
Append the last sentence to mention that some field/widget types have additional params, and link to the Widgets page for reference.